js/src/personal/haoking/Haoking.page.tsx

```diff
@@ -43,81 +57,107 @@ const darkTheme = createTheme({
 })
 
 export function HaokingPage() {
-  const getRootElement = () =>
-    typeof window === 'undefined' ? undefined : document.body
-
   return (
     <>
-      <MantineProvider
-        getRootElement={getRootElement}
-        theme={darkTheme}
-      />
-      <Flex
-        justify="flex-start"
-        direction="row"
+      <Box
+        bg={darkTheme.colors.black[4]}
+        h={rem('100vh')}
+        w={rem('100vw')}
       >
-        <Box
-          ml={90}
-          mt={75}
-          maw={600}
+        <Flex
+          justify="flex-start"
+          direction="row"
         >
-          <Title className={styles.title}>
+          <Box
+            ml={90}
+            mt={75}
+            maw={600}
+          >
+            <Title
+              ff="Helvetica, sans-serif"
+              className={styles.title}
+            >
+              <Text
+                inherit
+                variant="gradient"
+                component="span"
+                gradient={{
+                  from: darkTheme.colors.black[0],
+                  to: darkTheme.colors.black[1],
+                }}
+              >
+                {'Haoking Luo'}
+              </Text>
+            </Title>
+            <Title
+              fz={40}
+              c={darkTheme.colors.white[2]}
+            >
+              {'swe guy'}
+            </Title>
+            <Flex gap={15}>
+              <Button
+                leftSection={<IconBrandGithub size={16} />}
+                mt={15}
+                component="a"
+                href="https://github.com/luoh00"
+                target="_blank"
+                color={darkTheme.colors.black[0]}
+              >
+                {'GitHub'}
+              </Button>
+              <Button
+                leftSection={<IconBrandLinkedin size={16} />}
+                mt={15}
+                component="a"
+                href="https://github.com/luoh00"
+                target="_blank"
+                color={darkTheme.colors.black[0]}
+              >
+                {'LinkedIn'}
+              </Button>
+            </Flex>
+            <Image
+              p={15}
+              maw={375}
+              radius="xl"
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/3e/Raccoon_in_Central_Park_%2835264%29.jpg/1200px-Raccoon_in_Central_Park_%2835264%29.jpg"
+            />
+          </Box>
+          <Box
```

Comment:
This is getting a bit long. You want to start breaking these up into separate components and then importing them in.
Logically, it makes sense to encapsulate all of this text on the right into its own component.

Reply:
Will do for next PR